src/contracts/storage/tickmap.rs

```diff
@@ -25,6 +25,7 @@ impl Default for Tickmap {
     }
 }
 
+#[allow(dead_code)]
 pub fn get_max_chunk(tick_spacing: u16) -> u16 {
```

Comment:
can it be removed?

Reply:
We should keep this function as it's used in e2e tests. `#[allow(dead_code)]` is needed to remove the warning during the contract build